docs/markdown/Reference-tables.md

```diff
@@ -54,7 +54,7 @@ These are return values of the `get_linker_id` method in a compiler object.
 | optlink    | optlink (used with DMD)                     |
 | rlink      | The Renesas linker, used with CCrx only     |
 | xc16-ar    | The Microchip linker, used with XC16 only   |
-| ar2000     | The Texas Instruments linker, used with C2000 only |
+| ti-ar      | The Texas Instruments linker, used with C2000/MSP430 only |
```

Comment:
This is a problematic change because it breaks existing projects that use `ar2000` (and `c2000`) strings in their build files.

Reply:
Indeed, we likely need to keep the ar2000 name around for some time, it's probably going to have to be a case of:
```python
class TiCompiler:
   id = 'it-ar'

   < rest of implementation >

class ArCompiler(TiCompiler):
    is = 'ar200'
```

How to pass a deprecation message for ar2000 name is a more interesting question...

Reply:
Ah, I forgot about the ID backwards compat issue. I think for now I'll just modify the detection methods to assign that suggested wrapper class to the c2000 toolchain.

This would stop new users to the c2000 toolchain from being able to use 'ti' as their compiler ID, but it's not a huge userbase so I think it's ok.

One solution I can think of is allowing a compiler to have 2 IDs, but I think that generalisation is not worth the effort considering how niche this compatibility issue is.